app/models/micropost.rb

```diff
@@ -0,0 +1,19 @@
+class Micropost < ApplicationRecord
+  mount_uploader :picture, PictureUploader
+
+  belongs_to :user
+
+  validates :user_id, presence: true
+  validates :content, presence: true, length: {maximum: Settings.content_max_length}
+  validate  :picture_size
+
+  scope :order_by_created_at_desc, ->{order created_at: :desc}
```

Comment:
Bad naming. Name a scope depending on it functionality. As you are getting the micropost in descending order of created_at, name it like as by_latest/latest